ts/features/euCovidCert/screens/ko/EuCovidCertWrongFormatKoScreen.tsx

```diff
@@ -1,22 +1,124 @@
 import * as React from "react";
 import { connect } from "react-redux";
 import { Dispatch } from "redux";
-import { H1 } from "../../../../components/core/typography/H1";
+import { Image, StyleSheet } from "react-native";
+import { View } from "native-base";
 import { GlobalState } from "../../../../store/reducers/types";
 import { BaseEuCovidCertificateLayout } from "../BaseEuCovidCertificateLayout";
+import { H4 } from "../../../../components/core/typography/H4";
+import CopyButtonComponent from "../../../../components/CopyButtonComponent";
+import { EUCovidCertificateAuthCode } from "../../types/EUCovidCertificate";
+import { InfoScreenComponent } from "../../../../components/infoScreen/InfoScreenComponent";
+import wrongFormatImage from "../../../../../img/messages/error-message-detail-icon.png";
```

Comment:
Could you add also the red version of this image?
According to the design, the image must be red 
<img src=https://user-images.githubusercontent.com/26501317/121015536-65666600-c79b-11eb-8301-b76b09ee8e42.png width=300 />

Reply:
No problem for me, I used the other version in agreement with @thisisjp.

Reply:
Modified in [78e5b84](https://github.com/pagopa/io-app/pull/3093/commits/78e5b84850ee7b87a300e8e9e8938f41f6e6f918)
<img src="https://user-images.githubusercontent.com/11773070/121028853-6fdb2c80-c7a8-11eb-82aa-0c316981b868.png" width=300>